libs/ballot-interpreter-nh/src/js/mod.rs

```diff
@@ -116,6 +132,124 @@ pub fn interpret(mut cx: FunctionContext) -> JsResult<JsObject> {
     Ok(result_object)
 }
 
+pub fn find_layout(mut cx: FunctionContext) -> JsResult<JsObject> {
```

Comment:
Also, could this function be refactored to operate on one ballot image at a time? It doesn't seem like there are any dependencies between the two sides

Reply:
I opted to leave it as operating on a sheet rather than a single page since that's how it's used and it's easier to parallelize that way. If we wanted to make it work by page rather than sheet but still keep the speed we could change it to be `async` and then you could do an `await Promise.all(…)`.